monitoring/remote_monitoring_demo/recommendations_infra_demo/recommendations_demo/kruize/kruize.py

```diff
@@ -64,20 +64,18 @@ def create_experiment(input_json_file):
     print("\n************************************************************")
 
     # Validate the json
-    print("\nValidating the input json...")
-    isInvalid = validate_exp_input_json(input_json)
-    if isInvalid:
-        print(isInvalid)
-        print("Input Json is invalid")
-        exit(1)
-    else:
-        # read the json
-        print("\nCreating the experiment...")
-
-        url = URL + "/createExperiment"
-        response = requests.post(url, json=input_json)
-        print("URL = ", url, "   Response status code = ", response.status_code)
-        #print(response.text)
+    # COmmenting the validation as jsons are created through automation
```

Comment:
Even if the jsons are created through automation, this code is just going to check if it valid, why should we comment this out?